Project_0.Altis/OOP_Light/OOP_Light.h

```diff
@@ -350,10 +442,7 @@ private _oop_parents = []; \
 private _oop_methodList = []; \
 private _oop_newMethodList = []; \
 if (baseClassNameStr != "") then { \
-	if (!([baseClassNameStr, __FILE__, __LINE__] call OOP_assert_class)) then { \
-		missionNamespace setVariable [OOP_ERROR_DEBRIEFING_SECTION_VAR_NAME_STR, format ["Class %1 is not defined. File: %2", baseClassNameStr, __FILE__]]; \
-		endMission OOP_ERROR_DEBRIEFING_CLASS_NAME_STR; \
-	}; \
+	if (!([baseClassNameStr, __FILE__, __LINE__] call OOP_assert_class)) then {throw "invalid base class"}; \
```

Comment:
Who is catching it?

Reply:
Nobody, but it does exit the script immediately. Assert does the error surfacing itself right? We could certainly put catch block in init that would catch all of these if we wanted though.